### PR TITLE
Disable instrumentation pass with 3.0 configuration

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InstrumentationPass.cs [hunk 1]
```diff
@@ -9,6 +10,7 @@
 
 namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
+    [Obsolete("This type is obsolete and will be removed in a future version.")]
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InstrumentationPass.cs]: Can we just delete it in 3.0?

Comment (pranavkm) [reply src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InstrumentationPass.cs]: Removed

**Comment by pranavkm** [comment]
FYI @benaadams

**Comment by benaadams** [comment]
That works too 😄

**Pull request merged**